cms/templates/js/certificate-editor.underscore

```diff
@@ -28,6 +28,19 @@
                 <input id="certificate-course-title-<%= uniqueId %>" class="certificate-course-title-input input-text" name="certificate-course-title" type="text" placeholder="<%= gettext("Course title") %>" value="<%= course_title %>" aria-describedby="certificate-course-title-<%=uniqueId %>-tip" />
                 <span id="certificate-course-title-<%= uniqueId %>-tip" class="tip tip-stacked"><%= gettext("Specify an alternative to the official course title to display on certificates. Leave blank to use the official course title.") %></span>
             </div>
+
+            <div class="actual-course-number">
+                    <span class="certificate-label"><b><%= gettext('Course Number') %>: </b> </span>
+                    <span class="certificate-value"><%= course.get('num') %></span>
+            </div>
+
+            <% if (display_coursenumber) { %>
+                <div class="course-number-override">
+                    <span class="certificate-label"><b><%= gettext('Course Number Override') %>: </b></span>
+                    <span class="certificate-value"><%= display_coursenumber %></span>
```

Comment:
@saleem-latif Is it mention in the story that we need to display these data in certificate editor too ? I think it should be the part of certificate detail view.